main.go

```diff
@@ -43,8 +36,6 @@ func run(ctx context.Context) error {
 	signals := make(chan os.Signal, 1)
 	signal.Notify(signals, os.Interrupt, syscall.SIGTERM)
 
-	robotseo.Init(cmd.CmdFlagFields.RobotsFilePathReader)
-
 	// Run the service, providing an error channel for fatal errors
```

Comment:
You're still removing the init?

Reply:
Thought this will be added from dionisi's PR

Reply:
This line will be added again with next commit.